kiwi/system/uri.py

```diff
@@ -179,15 +179,16 @@ def is_remote(self):
             raise KiwiUriStyleUnknown(
                 'URI scheme not detected %s' % self.uri
             )
-        if uri.scheme in self.remote_uri_types:
+        if uri.scheme == 'obs' and Defaults.is_buildservice_worker():
+            return False
```

Comment:
Great! I like that solution, I think that now everything should work fine, I'll stay tuned, hopefully there won't be further surprises.